docs/app/Components/ComponentDoc/ComponentProps.js

```diff
@@ -23,7 +23,12 @@ export default class ComponentProps extends Component {
     }
   }
   defaultValueRenderer = (item) => {
-    const defaultValue = _.get(item, 'defaultValue.value')
+    let defaultValue = _.get(item, 'defaultValue.value')
+
+    if (_.startsWith(defaultValue, 'function ')) {
+      defaultValue = defaultValue.match(/^function(.*)\{/)[1].trim()
```

Comment:
Is this to get the function name? If I'm reading it correctly, seems like you could also do:

```js
if (typeof defaultValue === 'function') {
  defaultValue = defaultValue.name
}
```

I could be totally reading this wrong, so ignore if that's the case.